### PR TITLE
:sparkles: Cache template when fetched

### PR DESCRIPTION
**Pull request by ion098**

### CODE REVIEW

File: src/commands/new.rs [hunk 1]
```diff
@@ -60,11 +72,24 @@ pub async fn new(path: Utf8PathBuf, name: Option<String>) -> Result<(), CliError
...
 
     #[cfg(feature = "fetch-template")]
-    let template = match fetch_template().await {
-        Ok(bytes) => bytes,
+    let template: Vec<u8> = match fetch_template().await {
+        Ok(bytes) => {
+            if let Some(cache_file) = cached_template_path() {
+                info!("Storing fetched template in cache.");
+                fs::write(cache_file, &bytes).unwrap_or_else(|_| {
+                    info!("Could not cache template.");
```
Comment (Tropix126) [inline src/commands/new.rs:80]: I know that this isn't really your problem since the previous changes had this issue too, but these should be `error!`s

**Review by Tropix126 — CHANGES_REQUESTED** [review]

File: src/commands/new.rs (hunk 1 above)
Comment (Tropix126) [reply src/commands/new.rs:80]: Also the `info!` above should probably be a `debug!`

Comment (ion098) [reply src/commands/new.rs:80]: @Tropix126 wouldn't `warn!` make more sense compared to `error!`, since the code keeps running fine after?

**Review by Tropix126 — APPROVED** [review]
LGTM

**Pull request merged**